metadata-models/src/main/pegasus/com/linkedin/metadata/entity/DataProcessEntity.pdl

```diff
@@ -0,0 +1,31 @@
+namespace com.linkedin.metadata.entity
+
+import com.linkedin.common.DataPlatformUrn
+import com.linkedin.common.FabricType
+import com.linkedin.common.DataProcessUrn
+
+/**
+ * Data model for a Data Process entity
+ */
+record DataProcessEntity {
+
+  /**
+   * Urn for the Data Process
+   */
+  urn: DataProcessUrn
+
+  /**
+   * Data Process name(id)
+   */
+  name: optional string
+
+  /**
+   * Platform urn for the dataset in the form of urn:li:platform:{platform_name}
```

Comment:
This should be for data process right? and shouldn't it be orchestrator?

Reply:
Thanks for catching that. Fixed.